<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define Drink</string>
  </property>
  <widget class="QLabel" name="lb1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Define the drink</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Drink Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Drink Picture</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text &quot;Ingredient#1&quot; / 100ml</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text &quot;Ingredient#2&quot; / 100ml</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text &quot;Ingredient#3&quot; / 100ml</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text &quot;Ingredient#4&quot; / 100ml</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ql2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ql3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ql4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ql5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ml</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ml</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ml</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ml</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>440</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ql1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name Here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose file...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
